src/modules/projectContext.js

```diff
@@ -0,0 +1,41 @@
+import { ProjectImageContext } from './projectImageContext.js';
+
+class ProjectContext {
+
+    constructor(sessionManager, projectId) {
```

Comment:
javascript doc

Reply:
maybe also specify projectId has to be a uuid

Reply:
Added